src/bin/gitbot-auto-merge.js

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env node
+
+const program = require('commander');
+const auto_merge = require('../commands/auto-merge');
+
+program
+    .option('-O, --owner <owner>', 'Repository owner')
+    .option('-R, --repository-name <repository name>', 'Repository name')
+    .option('-N, --branch-number <branch number>', 'Branch number')
```

Comment:
... or is it the branch name? Would probably be easier to just provide the branch name?